app/src/main/java/org/simple/clinic/home/patients/illustration/HomescreenIllustration.kt

```diff
@@ -0,0 +1,29 @@
+package org.simple.clinic.home.patients.illustration
+
+import androidx.room.Dao
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import androidx.room.Query
+
+@Entity
+data class HomescreenIllustration(
```

Comment:
There's no migration added to create this table.

Reply:
Added.